gateway-ha/pom.xml

```diff
@@ -613,6 +590,29 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
```

Comment:
It would be nice to leave a code comment about the ordering.

Reply:
added! please check